src/Handler/CreateBoughtTogetherAssociationTypeHandler.php

```diff
@@ -15,11 +15,11 @@
 final class CreateBoughtTogetherAssociationTypeHandler
 {
     /**
-     * @param FactoryInterface<ProductAssociationTypeInterface> $productAssociationTypeFactory
+     * @implements FactoryInterface<ProductAssociationTypeInterface> $productAssociationTypeFactory
```

Comment:
why did we change it?

Reply:
Changing it back throws error 
![image](https://github.com/user-attachments/assets/4b1c2edc-5816-4f80-9685-d6d59320357c)